platforms/native/platform-native/src/integTest/groovy/org/gradle/nativeplatform/PlatformNativeComponentReportIntegrationTest.groovy

```diff
@@ -179,18 +177,20 @@ plugins {
     id 'assembler'
 }
 
+toolChains {
+    ${toolChain.buildScriptConfig}
+}
+
+flavors {
+    free
+    paid
+}
```

Comment:
❓ Why are `flavors` moved out of the model { } block? I understand that `toolchains` is used by the "new" native plugins, but flavors aren't, are they? So what's the value of pulling these out?

Reply:
Ack, bad copy and paste at the airport.

Reply:
I'm glad you noticed this because I just realized that the build type container and flavor container aren't needed by the non-software model plugins at all.  We have a separate BuildType class entirely.